### ISSUE TITLE
When seeing an unknown flag, it’s hard to tell the language

### ISSUE THREAD

**Issue by jiru** [Tatoeba/tatoeba2#2611]
**Story**
When I look at a group of sentences (main sentence + translations), I often see flags that I don’t know. I sometimes ask myself "I wonder what’s that language". From a computer, I can mouse-hover the flag to display the language name. That’s fine but I think it is rather difficult to guess for a newcomer. What’s more, I cannot do that from a phone. So the most obvious thing to do is to click on the (i) button to go to the sentence page. However, from there, again it is quite hard to find what’s the language: I have to look at the page title.

**Ideas**
1. On all sentence pages, clearly display the name of the language in which the sentence is.
2. On all flags, use modern tooltip that works by tapping on phones.

**Comment by githubshrek** [comment]
I'm looking at the problem here. I think there are a few things this leads me to observe.

Structurally, "the sentence" is shown as a single row in the same table as all the other sentences. This is actually somewhat confusing. For example, the sentence https://tatoeba.org/eng/sentences/show/9872646 is "Nie chcę z Panem mówić.", but when you look at the table, although it's the first row, and a bigger font, the table heading "Sentence #9872646" implies that all of the sentences in the table are simply versions of "Sentence #9872646", which is just not the case.

A second, related problem is the "menu" icons (Translate, Add to favourites, Add to list and the User profile link), which are separated from the table rows by a fine line. I guessed, wrongly, that these icons apply to all of the rows of the table. Not so. They only apply to the first row of the table.

I think therefore, the correct solution is to structurally separate the items which relate to the sentence itself, into a sentence details section:

* the sentence text, here "Nie chcę z Panem mówić."
* the sentence id, here 9872646
* **the sentence flag**, e.g. Polish
* **the sentence language [new]**
* the Translate icon
* the Add to favourites icon
* the Add to list icon
* the belongs to User link

from the sentence translations table, with multiple rows:

* sentence translation link arrow
* sentence translation flag, e.g. Romanian
* sentence translation text
* sentence translation copy icon
* sentence translation audio

Basically, we should clearly separate the elements which are part of the sentence and things which are not.

Another deeply related issue is the main point about the page is _not_ the sentence id. As IT people, it is extremely important, but for the general user, it's the sentence itself which is the main thing, and other things (flag, id, etc.) are merely attributes of the sentence itself.

**Comment by jiru** [comment]
Hi @githubshrek, welcome here and thank you for your valuable insight. :slightly_smiling_face: Since this is getting a bit off-topic, I answered you in a separate ticket: #2670.
